### ISSUE TITLE
EuroPython 2021

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#5943]
## about

https://ep2021.europython.eu

I will attend EP2021 (online) and will also be there for the sprints.

To take part in the sprints, you need at least the for-free "sprint-only ticket", see the EP site.

There is also a for-free "stream-only ticket", btw and of course the normal paid conference ticket.

## sprints
Topics:
- whatever you want it to be
- get an introduction into the codebase
- questions **about the code / project**
- fix some bugs
- improve docs
- improve compatibility / portability, e.g. #4771
- work on some near-future milestone: https://github.com/borgbackup/borg/milestones
- work on https://github.com/borgbackup/borg-import
- work on https://github.com/borgbase/vorta (GUI for borgbackup)
- check this issue tracker for issue you'll like to work on. There are tags for [easy](https://github.com/borgbackup/borg/issues?q=is%3Aissue+is%3Aopen+label%3Aeasy), [help wanted](https://github.com/borgbackup/borg/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and more.

Preparations:
- make sure you have a decent, fast network connection [use wired LAN if possible]
- to avoid bad audio, use a headset or at least headphones/earphones, **not** speakers
- have a python 3.5+ based development environment ready
- configure your editor to indent by 4 spaces (not tabs) and to use lf-only for line ends (not cr/lf)
- read/follow: https://borgbackup.readthedocs.io/en/stable/installation.html#from-source
- read https://borgbackup.readthedocs.io/en/stable/development.html
- read https://borgbackup.readthedocs.io/en/stable/internals.html (optional, but helpful)

## open space

No borg talk or session are planned yet, but if there is interest, we could have some open space borgbackup users meeting / Q&A session. Conf room 6 "Silly Walks" is for Open Space sessions.

## interested?

If that sounds interesting, speak up below and tell about your borg europython wishes / plans.

**Comment by ThomasWaldmann** [comment]
For after the EP sprints, please use our normal communication channels:

github, IRC chat, mailing list - you find them all via www.borgbackup.org.

**Comment by ThomasWaldmann** [comment]
EP is over.

2 developers showed interest into the sprints, 1 actually worked on something and did a PR (plus me, I worked on the macOS unicode ticket).

Guess I will have to consider whether there will be future EP sprints. It's just too little interest / community involvement.

Also, if somebody wants to contribute, there is actually no need to wait for such an event, one can contribute at any time and I can also offer what I posted above at any time, just ask.
